sysbuild/Kconfig.appcore

```diff
@@ -0,0 +1,89 @@
+# Copyright (c) 2024 Nordic Semiconductor
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+
+config SUPPORT_APPCORE
+	bool
+	default y if (SOC_NRF5340_CPUNET || SOC_NRF54H20_CPURAD)
```

Comment:
I am not sure if this should be enabled bu default for nRF54h20. It is mandatory for nRF53 because netcore is enabled by application core so app core needs to run any application always

Reply:
Good catch, but I think the `SUPPORT_APPCORE` is fine. The `APPCORE` choice should have a default for APPCORE_NONE in case of nRF54.